### DISCUSSION TITLE
Request to Add Email Address (@cdtn.br) for Access to the Server

### DISCUSSION THREAD

**Discussion by danilorcf** [RosettaCommons/rosetta#81] [ROSIE]
Dear ROSIE Support Team,
I hope this message finds you well.
I am writing to inform you that my email address with the domain "@cdtn.br" belongs to an institute of research called "Centro de Desenvolvimento da Tecnologia Nuclear" (Center for the Development of Nuclear Technology), which is located in Minas Gerais, Brazil. I am currently a doctoral student at this institute.
I would appreciate it if you could add my email address to your system to grant me access to the server.
Thank you for your attention to this matter. Please let me know if you require any further information.
Best regards,
Danilo Ferreira
Doctoral Student, Sciences and Technology of Materials, Radiations, and Minerals
Centro de Desenvolvimento da Tecnologia Nuclear
Brazil

**Comment by lyskov** [comment]
Thank you for letting us know! This should be fixed now. Please feel free to re-open if you are still having issues. Thanks,
